### ISSUE TITLE
how does freelan compare with other solutions

### ISSUE THREAD

**Issue by leggewie** [freelan-developers/website#6]
It would be nice to have some kind of comparison with other VPN software. Why is freelan "better" and how?

- openvpn
- n2n
- gvpe
- hamachi
- tinc
- ...

Let's discuss about the format first.

**Comment by ereOn** [comment]
I wanted to avoid speaking about other solutions because well... I wouldn't want anyone to speak half-truth on freelan either.

Even if we come up with a fair comparison, those softwares evolve as freelan do (and perhaps even more these days) and whatever we wrote becomes stale and has to be maintained.

I like the idea of exposing facts we know for sure about freelan and let people decide which solution is "superior" (or better yet: best suited for their need). My goal with freelan has never been world domination or even money: I wanted to solve a problem I had and learn something in the process.

Don't get me wrong, I love that people use my product but I don't want to cheat anyone with a false, unfair or outdated comparison.

Does that make sense ?

**Comment by s-vincent** [comment]
I agree with @ereOn, we should stay objective and focus to FreeLAN not on alternative solutions. 

For the website, it will be interresting to have an board/array with the different features and a little "check" symbol saying "supported by FreeLAN". So potential users can see if their use-cases is listed.

**Comment by leggewie** [comment]
Thank you for your input.  I apologize for the delay in responding.

Some good points.  I didn't necessarily mean this as advertisement copy.  I meant it as a reminder to myself of why I got involved with freelan in the first place (or why I assume you started coding it).  In other words, what was missing from the solutions that came closest?  In my case, I wanted a VPN to do VoIP.  My goal isn't necessarily uber-security, just good enough ;)  I did not want having to set up a central server or have to trust somebody else's central server (that rules out the main "competitor" OpenVPN).  It seemed that I needed UDP support and that ruled out a few others.  I ended up going with n2n which has served me well and which I maintain in Debian but which is dead upstream and has been dead for quite a while.  I had some trouble with getting routing to work (I want edge routers to combine dispersed LANs remotely).  I believe (but can't be sure anymore) this and the dead upstream is what initially drew me to freelan.  Thus my request to [slim things down for embedded use](freelan-developers/freelan#152).  n2n needs a central relay for NAT clients, but the relay server cannot decrypt or inspect any traffic it relays.  The clients do not have to trust the relay at all ([this is something I hope we will have in freelan eventually as well](freelan-developers/freelan#54)).  Tinc was another solution that I considered and that came close, but IIRC, it fell short on one requirement.  I cannot remember which anymore at the moment which is what made me open this ticket, so that I would document it and be able to revisit my reasoning in the future.  That's all.

Maybe a better solution would be testimonials.  People could say "I chose freelan because ...  XYZ was close but unsuitable for me because ..."  Or maybe that's something for personal blogs (mine has been defunct for years ;-)

**Comment by ereOn** [comment]
@leggewie That makes a lot of sense. I would definitely go with testimonials over a possibly unfaire/biased comparison. That's a good suggestion, thanks !
